cg/services/run_devices/pacbio/run_validator/pacbio_run_validator.py

```diff
@@ -0,0 +1,42 @@
+from pathlib import Path
+
+from cg.constants.constants import FileFormat
+from cg.services.decompression_service.decompressor import Decompressor
+from cg.services.run_devices.abstract_classes import RunValidator
+from cg.services.run_devices.pacbio.run_data_generator.run_data import PacBioRunData
+from cg.services.run_devices.pacbio.run_file_manager.models import PacBioRunValidatorFiles
+from cg.services.run_devices.pacbio.run_file_manager.run_file_manager import PacBioRunFileManager
+from cg.services.validate_file_transfer_service.validate_file_transfer_service import (
+    ValidateFileTransferService,
+)
+
+
+class PacBioRunValidator(RunValidator):
+    """PacBio run validator.
+    Ensure that the run can start post-processing of a pacbio run can start.
+    """
+
+    def __init__(
+        self,
+        decompressor: Decompressor,
+        file_transfer_validator: ValidateFileTransferService,
+        file_manager: PacBioRunFileManager,
+    ):
+        self.decompressor = decompressor
+        self.file_transfer_validator = file_transfer_validator
+        self.file_manager = file_manager
+
+    def ensure_post_processing_can_start(self, run_data: PacBioRunData):
+        """Ensure that a post processing run can start."""
```

Comment:
Since this function is not named `validate_something` I think we should state that it raises an error if the pre-condition is not fulfilled and also decompresses a reports folders

Reply:
Errors are raised elsewhere. I added a clarification of what ensuring a run can start includes.